### PR TITLE
Changes to security considerations

### PR DESCRIPTION
**Pull request by theri**
Based on #93, clarify why confidentiality etc are out of scope.

### CODE REVIEW

**Review by boucadair — APPROVED** [review]

File: draft-irtf-panrg-path-properties.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ This may diminish the usefulness of path-aware technologies over this path.
 Security related properties such as confidentiality and integrity protection of payloads are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
 Likewise, properties for trust relations between entities cannot be meaningfully defined without a concrete threat model, and defining a threat model is out of scope for this draft.
 Properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document.
-Such properties are tied to the communicating nodes and the protocols they use (e.g., client and server using HTTPS, or client and remote network node using VPN) while the path is typically oblivious to them.
+Such properties are tied to the communicating nodes and the protocols they use (e.g., client and server using HTTPS, or client and remote network node using VPN) as well as additional context such as keying material and who has access to this context. In constrast, the path as defined in this document is typically oblivious to these aspects.
```
Comment (boucadair) [inline draft-irtf-panrg-path-properties.md:323]:
```suggestion
Such properties are tied to the communicating nodes and the protocols they use (e.g., client and server using HTTPS, or client and remote network node using a VPN service) as well as additional context, such as keying material and who has access to such a context. In contrast, the path as defined in this document is typically oblivious to these aspects.
```

**Comment by theri** [comment]
I pushed more changes as discussed in issue #93.
@cyrill-k, I remember you came up with much of the language that I modified/framed in this PR. Please let me know your thoughts, ideally by Friday March 3rd (PT). Thank you!

File: draft-irtf-panrg-path-properties.md [hunk 2]
```diff
@@ -311,16 +311,18 @@ Metrics such as loss patterns {{RFC3357}} and loss episodes {{RFC6534}} can be e
...
 Such information could then allow network operators to block or re-prioritize traffic for certain users and/or application.
 Conversely, if privacy enhancing technologies, e.g., MASQUE proxies [RFC9298], are used on a path, the path may only be partially visible to any single entity.
 This may diminish the usefulness of path-aware technologies over this path.
-Security related properties such as confidentiality and integrity protection of payloads are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
+
+The need for, and potential definition of, security and privacy related path properties, such as confidentiality and integrity protection of payloads, are the subject of ongoing discussion and research, e.g., see {{?RFC9049}} and {{?RFC9217}}. As the thinking around such properties is not mature enough, such properties are out of scope for this document.
```
Comment (cyrill-k) [inline draft-irtf-panrg-path-properties.md:321]: Some rewording suggetions:

```suggestion
The need for, and potential definition of, security and privacy related path properties, such as confidentiality and integrity protection of payloads, are the subject of ongoing discussion and research, e.g., see {{?RFC9049}} and {{?RFC9217}}. As the discussion of such properties is not mature enough, they are out of scope for this document.
```

File: draft-irtf-panrg-path-properties.md [hunk 3]
```diff
@@ -311,16 +311,18 @@ Metrics such as loss patterns {{RFC3357}} and loss episodes {{RFC6534}} can be e
...
+One aspect discussed in this context is that security related properties are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
 Likewise, properties for trust relations between entities cannot be meaningfully defined without a concrete threat model, and defining a threat model is out of scope for this draft.
```
Comment (cyrill-k) [inline draft-irtf-panrg-path-properties.md:323]: Let's use `document` instead of `draft` consistently.

```suggestion
Likewise, properties for trust relations between entities cannot be meaningfully defined without a concrete threat model, and defining a threat model is out of scope for this document.
```

File: draft-irtf-panrg-path-properties.md [hunk 4]
```diff
@@ -311,16 +311,18 @@ Metrics such as loss patterns {{RFC3357}} and loss episodes {{RFC6534}} can be e
...
-Properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document.
-Such properties are tied to the communicating nodes and the protocols they use (e.g., client and server using HTTPS, or client and remote network node using VPN) while the path is typically oblivious to them.
+In addition, we suspect that properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document.
```
Comment (cyrill-k) [inline draft-irtf-panrg-path-properties.md:324]: I'm not too happy about the word "suspect", but I can't come up with a more fitting word.
Maybe we can combine it with the next sentence and rephrase it to: "Properties related to confidentiality, integrity, and trust seem to be orthogonal to the path terminology and path properties defined in this document, since they are tied ..."
What do you think?

**Review by cyrill-k — COMMENTED** [review]
Generally this looks good to me.
I just have some suggestions for rephrasing.

File: draft-irtf-panrg-path-properties.md (hunk 4 above)
Comment (theri) [reply draft-irtf-panrg-path-properties.md:324]: That sounds good to me, I'll apply your change.

**Review by cyrill-k — APPROVED** [review]
